### DISCUSSION TITLE
sqlite not selectable as a database adapter

### DISCUSSION THREAD

**Discussion by logkirk** [xenocrat/chyrp-lite#263] [Q&A] [answered]
I'm following the instructions in [the quick start guide](https://chyrplite.net/wiki/Quick-Start-Guide.html). I have deployed the chyrp code to my public directory on my site. When first loading the installer, I get warnings about write access, so I used `chown` to change ownership of the chyrp directory to the apache webserver user. After this, the warnings go away.

In the "Adapter" dropdown, only MySQL is listed, even though I have SQLite installed:

```
SQLite version 3.37.2 2022-01-06 13:25:41
```

What do I need to do to enable the SQLite adapter?

**Comment by xenocrat** [comment, marked as answer]
Hello there,

Your build of PHP does not have the [SQLite PDO driver](https://www.php.net/manual/en/ref.pdo-sqlite.php) enabled, or the driver is enabled but not able to function. If you have administrative control of your PHP installation, you can follow the instructions in the link to enable support. At first glance it seems the problem may be that your installed SQLite version is to old to work with PHP: SQLite 3.5.0+ is required.

**Reply by logkirk** [reply]
As you suggested, I did not have the correct driver installed. I got it installed with `sudo apt install php8.1-sqlite3`. Thanks!
